lib/airbrake/capistrano/tasks.rb

```diff
@@ -6,7 +6,8 @@
   namespace :airbrake do
     desc "Notify Airbrake of the deploy"
     task :deploy do
-      on roles(:all) do
+      role = roles(:all, select: :primary).first || roles(:all).first
```

Comment:
Have you tested this? It looks like the `on` method expects an Array (and we now pass a single item)

* https://github.com/capistrano/capistrano/blob/d947b25aa2f788a9cf083ab3d8c2e92675507740/lib/capistrano/dsl.rb#L52-L53
* https://github.com/capistrano/capistrano/blob/e30516859878e510c14f24bfbe14abe94b20bd03/lib/capistrano/configuration.rb#L124